doc/reference/dune/alias.rst

```diff
@@ -1,55 +1,58 @@
+.. highlight:: dune
+
 alias
 -----
 
-The ``alias`` stanza adds dependencies to an alias or specifies an action to run
-to construct the alias.
+.. describe:: (alias ...)
+
+   Add dependencies to an alias.
```

Comment:
The wording here is a bit unclear whether or not you can create new aliases or can just add dependencies to existing ones. I think it would be better to preempt the users question and state this here directly.